### PR TITLE
Sm/postorgcreate-hook

### PR DESCRIPTION
**Pull request by mshanemc**
### What does this PR do?
fix#634 postorgcreate hook becomes async

### What issues does this PR fix or reference?
@W-11572657@ 
https://github.com/forcedotcom/cli/issues/1700

### CODE REVIEW

File: src/util/getJwtAudienceUrl.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import { SfdcUrl } from './sfdcUrl';
 export async function getJwtAudienceUrl(options: OAuth2Config & { createdOrgInstance?: string }): Promise<string> {
   if (options.loginUrl) {
     const url = new SfdcUrl(options.loginUrl);
-    return await url.getJwtAudienceUrl(options.createdOrgInstance);
```
Comment (cristiand391) [inline src/util/getJwtAudienceUrl.ts:14]: Why is this needed?

Comment (mshanemc) [reply src/util/getJwtAudienceUrl.ts:14]: it's unrelated to the PR.

**Review by cristiand391 — APPROVED** [review]
